### DISCUSSION TITLE
Wrong result in date diff in days

### DISCUSSION THREAD

**Discussion by babu-zolve** [moment/moment#6416] [Q&A] [answered]
**example 1:**
  var a = moment([2024, 2, 23]);
  var b = moment([2023, 2, 24]);
  console.log(a.diff(b, 'days') );
  **Output : 365**
  **example 2:**
  var a = moment([2024, 2, 24]);
  var b = moment([2024, 2, 23]);
  console.log(a.diff(b, 'days') );
  **Output : 1**
  
  As we see in example 2, this is not including the both dates for diff in term of days.  So ideally in first example, it should give 364(5+31+30+31+30+31+31+30+31+30+31+31+22) days.

**Comment by utf26** [comment, marked as answer]
months are 0-indexed in moment.js (0 = January, 1 = February, etc.)
